src/plugins/kvantum.py

```diff
@@ -10,9 +10,13 @@ def available_themes(self) -> dict:
         if not self.available:
             return {}
 
-        path = '/usr/share/Kvantum'
-
-        themes = get_stuff_in_dir(path, type='dir')
+        paths = ['/usr/share/Kvantum',str(Path.home()) + '/.config/Kvantum']
```

Comment:
Seems like a whitespace is missing between the two paths in the list

Reply:
This will be fixed.